packages/fxa-settings/src/pages/Signin/SigninBounced/index.tsx

```diff
@@ -41,13 +42,10 @@ const SigninBounced = ({
 
   return (
     <>
-      <div className="mb-4">
-        <FtlMsg id="signin-bounced-header">
-          <h1 className="card-header">
-            Sorry. We’ve locked your{'\u00A0'}account.
```

Comment:
This non-breaking space was originally added [here](https://github.com/mozilla/fxa-content-server/pull/5318), looks like because the header used to drop on two lines and we didn't like "account" being on its own line. Not applicable now 🤷‍♀️

Reply:
FWIW (probably not much) the Ftl string also has this non-breaking space. Probably fine to leave as is (vs updating and changing the id)

Reply:
Awh man you're right / good thinking. I wonder if this will break the l10n test when enabled...